benchmarks/graphics_pipeline/main.cpp

```diff
@@ -534,6 +560,8 @@ void ProjApp::LoadScene(
 
         PPX_ASSERT_MSG(data->buffers_count == 1, "Only supports one buffer for now.");
         PPX_ASSERT_MSG(data->buffers[0].data != nullptr, "Data not loaded. Was cgltf_load_buffer called?");
+
+        PPX_ASSERT_MSG(data->textures_count <= kMaxMaterialCount , "No support for more than " << kMaxMaterialCount  << " distinct materials");
```

Comment:
Is it correct in using `data->textures_count`? It shouldn't be using `data->materials_count`?